jinahub/encoder/image/clip.py

```diff
@@ -0,0 +1,22 @@
+import clip
+import torch
+import numpy as np
+from jina import Executor, DocumentArray, requests
+
+
+class CLIPImageEncoder(Executor):
+    """Encode image into embeddings."""
+
+    def __init__(self, model_name: str = 'ViT-B/32', *args, **kwargs):
```

Comment:
should have a 'default_traversal_path' too

Reply:
thanks, added